cypress/support/step-definitions/common-steps.js

```diff
@@ -274,8 +275,8 @@ Then('fieldHelp on preview is set to {word} in NoIFrame', (text) => {
 });
 
 Then('{string} fieldHelp on preview is set to {word}', (position, text) => {
-  cy.wait(1500, { log: DEBUG_FLAG }); // delayed to ensure it to run on CI
-  fieldHelpPreviewByPosition(positionOfElement(position)).should('have.text', text);
+  // cy.wait(1500, { log: DEBUG_FLAG }); // delayed to ensure it to run on CI
```

Comment:
if we are not using this line of code - please delete it.

Reply:
fixed